electron/src/update/squirrel.ts

```diff
@@ -104,46 +97,17 @@ async function spawnUpdate(args: string[]): Promise<void> {
   }
 }
 
-function createShortcut(location: string): boolean {
-  // As documented in https://github.com/electron/windows-installer/issues/296,
-  // Squirrel has problems with notification clicks on Windows 10.
-  // The easiest workaround is to create shortcuts on our own.
```

Comment:
Looks like we're not doing anything to resolve that issue anymore, something changed?

Reply:
We did retry to go native and avoid custom code. It seems that the problem is no longer existing... Will give it a few more tries and I'm planning on asking QA to focus on this topic as well :)